Makefile

```diff
@@ -146,7 +146,7 @@ docker-pytest:
 	@echo "Creating container..."
 	@docker run --name pytest-estimators -d -it ${image_name} sh \
 		&& docker exec pytest-estimators sh -c " \
-		pytest --doctest-modules -n auto -vv \
+		pytest --doctest-modules -n auto -vv -s \
```

Comment:
no test captures?

Reply:
Yes, we now need some manual printing to keep track of failing KATs.